stock_picking_mgmt_weight/views/stock_picking_views.xml

```diff
@@ -0,0 +1,88 @@
+                <field name="date_tare" invisible="1"/>
+                <field name="gross_weight" attrs="{'invisible': [('type_scale', '=', False)]}"/>
+                <field name="date_gross_weight" invisible="1"/>
+                <field name="net_weight" readonly="0" attrs="{'invisible': [('type_scale', '=', False)]}"/>
+            </xpath>
+            <xpath expr="//form/group/field[@name='description_picking']" position="after">
+                <field name="type_scale" invisible="1"/>
+                <field name="theoretical_qty" attrs="{'invisible': [('type_scale', '=', False)]}"/>
+                <field name="exclude_tare" attrs="{'invisible': [('type_scale', '=', False)]}"/>
+                <field name="tare" force_save="1" attrs="{'readonly': [('exclude_tare', '=', True)]}"/>
+                <field name="date_tare" invisible="1"/>
+                <field name="gross_weight" attrs="{'invisible': [('type_scale', '=', False)]}"/>
+                <field name="date_gross_weight" invisible="1"/>
+                <field name="net_weight" readonly="0" attrs="{'invisible': [('type_scale', '=', False)]}"/>
+            </xpath>
+            
+        </field>
+    </record>
+    
+    <record id="view_stock_picking_weight_tree_inherit" model="ir.ui.view">
+        <field name="name">stock.picking.tree (weight)</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.vpicktree"/>
+        <field name="groups_id" eval="[(4,ref('vehicle.group_vehicle_user'))]" />
+        <field name="arch" type="xml">
+            <xpath expr="//tree/field[@name='activity_exception_decoration']" position="after">
+                <field name="vehicle_id" optional="show"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_stock_picking_weight_search_inherit" model="ir.ui.view">
+        <field name="name">stock.picking.search (weight)</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_internal_search"/>
+        <field name="groups_id" eval="[(4,ref('vehicle.group_vehicle_user'))]" />
+        <field name="arch" type="xml">
+            <xpath expr="//search/field[@name='picking_type_id']" position="after">
+                <field name="vehicle_id"/>
+            </xpath>
+        </field>
+    </record>
+    <!-- New Stock Move Line -->
+
+    <record id="view_stock_picking_weight_form_inherit2" model="ir.ui.view">
+        <field name="name">stock.picking.form (move line)</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//header/button[@name='action_cancel']" position="before">
+                <button name="%(action_report_picking_tag)d" string="Print Tag" type="action" groups="base.group_user"/>
```

Comment:
This button should be placed in `stock.picking.form (weight)` view, together with the other scale additions, and also only shown if the picking is a scale type one.